MtApi5/MtApi5Client.cs

```diff
@@ -475,6 +475,17 @@ public bool OrderCloseAll()
         {
             return sendCommand<bool>(Mt5CommandType.OrderCloseAll, null);
         }
+
+        ///<summary>
+        ///Closes a position with the specified ticket.
+        ///</summary>
+        ///<param name="ticket">Ticket of the closed position.</param>
+        public bool PositionClose(int ticket)

```

Comment:
I didn't use the ulong type because the value never came right on the side of MetaTrader.